backend/schema.js

```diff
@@ -14,71 +15,151 @@ const typeDefs = gql`
   }
 
   type Team {    
-    Id: ID!
-    Name: String!    
-    Header: String
-    Description: String
-    CreatedAt: Date!
-    CreatedByUserId: ID!
-    UpdatedAt: Date
-    UpdatedByUserId: ID
-    Members: [Member]
-    JobsAppliedTo: [Job]
-    Tags: [ScoredTag]
+    id: ID!
+    name: String!    
+    header: String
+    description: String
+    createdAt: Date!
+    createdByUserId: ID!
+    updatedAt: Date
+    updatedByUserId: ID
+    members: [Member]
+    jobsAppliedTo: [Job]
+    tags: [ScoredTag]
   }
 
   type Member {
-    User: User!
-    Role: RoleType!
-    NoticePeriod: Int
-    AvailableAt: Date
+    user: User!
+    role: RoleType!
+    noticePeriod: Int
+    availableAt: Date
   }
 
   type User {
-    Id: ID!
-    Available: Boolean!
-    FirstName: String! 
-    LastName: String! 
-    Description: String! 
-    SocialAccounts: [SocialAccount]
-    CreatedAt: Date!
-    CreatedByUserId: ID!
-    UpdatedAt: Date
-    UpdatedByUserId: ID
-    TeamMemberOf: [Team]
-    JobsCreated: [Job]
-    Tags: [ScoredTag]
+    id: ID!
+    available: Boolean!
+    firstName: String! 
+    lastName: String! 
+    description: String! 
+    socialAccounts: [SocialAccount]
+    createdAt: Date!
+    createdByUserId: ID!
+    updatedAt: Date
+    updatedByUserId: ID
+    teamMemberOf: [Team]
+    jobsCreated: [Job]
+    tags: [ScoredTag]
   }
 
   type Job {
-    Id: ID!
-    Name: String!
-    Header: String
-    Description: String
-    Applications: [Team]
-    CreatedAt: Date!
-    CreatedByUserId: ID!
-    UpdatedAt: Date
-    UpdatedByUserId: ID
-    Tags: [Tag]
+    id: ID!
+    name: String!
+    header: String
+    description: String
+    applications: [Team]
+    createdAt: Date!
+    createdByUserId: ID!
+    updatedAt: Date
+    updatedByUserId: ID
+    tags: [Tag]
   }
 
   type SocialAccount {
-    ExternalId: String!
-    Type: SocialAccountType!
+    externalId: String!
+    type: SocialAccountType!
   }
  
   type ScoredTag {
-    Name: String!
-    Score: Int
+    name: String!
+    score: Int
   }
 
   type Tag {
-    Name: String!
+    name: String!
   }
 
-  type Query {
-    allUsers: [User]
+  #QUERIES#####################################
+
+  #based on token
+  query {
+    me: User
+  }
+
+  query Team($id: Int) {
+    team(id: $id): Team
```

Comment:
You can write all queries in a single object like:
See https://github.com/apollographql/apollo-server-tutorial/blob/master/data/schema.js

```
query {
    me: User
    team(id: $id): Team
    ...
  }
```